spec/salestation/web/responses/unprocessable_entity_from_schema_errors_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'spec_helper'
+
+describe Salestation::Web::Responses::UnprocessableEntityFromSchemaErrors do
+  subject(:error) { described_class.create(attributes) }
+  let(:attributes) { {
+    errors: errors,
+    hints: hints,
+  } }
+
+  let(:errors) { {} }
+  let(:hints) { {} }
+
+  context 'error with one message' do
+    let(:errors) { {content: ['is missing']} }
+    let(:hints) { {content: ['is missing']} }
+
+    it 'parses error message' do
+      expect(error.message).to eq("'content' is missing")
+    end
+
+    it 'parses error debug message' do
+      expect(error.debug_message).to eq("'content' is missing")
+    end
+  end
+
+  context 'error with multiple message' do
+    let(:errors) { {content: ['is missing', 'is invalid']} }
+    let(:hints) { {content: ['is missing', 'is invalid']} }
+
+    it 'parses error message' do
+      expect(error.message).to eq("'content' is missing and is invalid")
+    end
+
+    it 'parses error debug message' do
+      expect(error.debug_message).to eq("'content' is missing and is invalid")
+    end
+  end
+
+  context 'error with nested errors' do
+    let(:errors) { {message: {'content'=>['is missing']}} }
+    let(:hints) { {message: {'content'=>['is missing']}} }
+
+    it 'parses error message' do
+      expect(error.message).to eq("'message' => { 'content' is missing }")
```

Comment:
Without extra quotes:
> {"message":"'site_id' must be filled. 'engagement_id' must be filled. 'operator_id' must be filled. 'some_nested_field' => { 'id' is missing. 'content' is missing }"

I would still prefer `... must be filled. 'message.id' is missing. 'message.content' is missing` It is more human readable that nested hashes in a string.